### DISCUSSION TITLE
Error 78 when compiling simple C program with cosmocc

### DISCUSSION THREAD

**Discussion by neezer** [jart/cosmopolitan#1250] [Q&A]
👋🏻 Hi all, new to cosmo & C programming in general, so please bear with me.

I'm trying to compile [this simple program](https://stackoverflow.com/a/17371925) with `cosmocc`

```sh
./bin/cosmocc -lm -o time main.c
```

And I'm greeted with the following error:

> .ape-1.10[42762]: pinsyscalls addr 7f0013e0 code 5, pinoff 0xffffffff (pin 0 0-0 0) (libcpin 0 0-0 0) error 78
> Abort trap (core dumped)

I don't see a core dump file anywhere, otherwise I'd post that (`kern.nosuidcoredump -> 0`, if that matters).

My program compiles fine using:

```sh
cc -lm -o time main.c
```

I'm using `cosmocc` from `cosmocc-3.6.2.zip` from the main website. I'm running OpenBSD 7.5 (current).

Anyone know what's up with this?

**Comment by neezer** [comment]
Maybe this an issue with APE on OpenBSD 7.5 (current) in general? I just tried running the latest RedBean server on my box and got a nearly identical error:

> .ape[24517]: pinsyscalls addr 200417 code 33, pinoff 0xffffffff (pin 0 0-0 0) (libcpin 0 0-0 0) error 78

**Comment by G4Vi** [comment]
Are you able to build on Linux and run on OpenBSD? I think Linux is our only supported development platform, but fixes welcome!

**Reply by neezer** [reply]
I think [my experience trying to run redbean](https://github.com/jart/cosmopolitan/discussions/1250#discussioncomment-10269614) would argue against "build on Linux/run on OpenBSD" working. 🫤

**Reply by G4Vi** [reply]
Sorry, I missed that. To get greater visibility, I made an issue https://github.com/jart/cosmopolitan/issues/1263
